### ISSUE TITLE
Album listings do not respect the Pagination setting

### ISSUE THREAD

**Issue by ToxicFrog** [airsonic-advanced/airsonic-advanced#399]
## Problem description

When viewing an individual album, the Pagination setting is respected, i.e. if I set it to -1 it will display all tracks. This doesn't seem to happen for views that display a list of albums, though, or a list of directories containing albums -- it resets to 7 every time one of these pages is loaded.

### Steps to reproduce

Set pagination to `-1` under Settings.
View an album. Observe that the `Show __ entries` dropdown is set to `All` and all tracks are displayed.
Now view a list of albums or directories (e.g. by clicking on one of the genre selectors on the left sidebar). Observe that it says `Show 7 entries` and, indeed, only seven are displayed.

## System information

 * **Airsonic version**: 10.6.0-SNAPSHOT.20200812000800; this is actually 10.6.0-SNAPSHOT.20200424035525 with a local patch applied to make podcasts sort in chronological order.
 * **Operating system**: NixOS
 * **Java version**: openJDK 14.0.2
 * **Proxy server**: nginx 1.18.0
 * **Client**: Firefox 70 (but also affects Chrome/Chromium)
 * **Language**: en

**Comment by randomnicode** [comment]
Albums/subdirectories are hardcoded to 7, is there a strong sentiment here to make them adhere to the rest of the table settings?

It was hardcoded for visual aesthetics, but we can change that

**Comment by randomnicode** [comment]
> local patch applied to make podcasts sort in chronological order.

Mind submitting a PR to this fork so everyone can benefit?

**Comment by ToxicFrog** [comment]
For my part there is basically no occurrence where I only want it to show 7 entries -- my screen will comfortably show 20+, and in general I'd rather scroll through a long list than repeatedly click through a bunch of shorter ones. I don't know if anyone else feels as strongly.

> Mind submitting a PR to this fork so everyone can benefit?

The patch is very quick and dirty (in particular, there is no configuration setting for it), and historically I have observed that "oldest episode first" vs. "newest episode first" is a point of strong contention (and one I appear to be in the minority in in preferring oldest-first), so I figured there wasn't much point in sharing it without configuration knobs, which I don't currently have time to write.

For those interested, [here is the patch](https://gist.github.com/ToxicFrog/28eb2f70c0a34c8049c9797df2b833c7).
